server/utils/courseUtils.test.ts

```diff
@@ -35,19 +40,21 @@ describe('courseUtils', () => {
         prerequisiteSummaryListRows: [
           {
             key: { text: 'Setting' },
-            value: { text: course.coursePrerequisites[3].description },
+            value: { text: settingPrerequisite.description },
           },
           {
             key: { text: 'Gender' },
-            value: { text: course.coursePrerequisites[0].description },
+            value: { text: genderPrerequisite.description },
           },
           {
             key: { text: 'Risk criteria' },
-            value: { text: course.coursePrerequisites[2].description },
+            value: {
+              text: `${riskCriteriaPrerequisites[0].description}, ${riskCriteriaPrerequisites[1].description}`,
```

Comment:
I think this would serve as a useful piece of documentation if we made it its own test e.g. `"It concatenates prerequisites with the same name"`